src/components/Portfolio.js

```diff
@@ -5,14 +5,30 @@ export default function Portfolio() {
         <div className="portfolio--container">
             <h2 className="portfolio--title">Portfolio</h2>
             <div className="portfolio--link-container">
-                <a href="https://shimmering-malasada-04a43c.netlify.app" target="_blank" rel="noopener noreferrer" className="portfolio--link">
-                    React Notes App
-                </a>
-                <button>
-                    <a href="https://github.com/nngrz/React-project-set/tree/main/notes_app" target="_blank" rel="noopener noreferrer" className="portfolio--btn">
-                        Source Code
+                <div className="projects">
+                    {/* project--React Notes App */}
+                    <a href="https://shimmering-malasada-04a43c.netlify.app" target="_blank" rel="noopener noreferrer" className="project--link">
+                        React Notes App
                     </a>
-                </button>
+                    <button>
+                        <a href="https://github.com/nngrz/React-project-set/tree/main/notes_app" target="_blank" rel="noopener noreferrer" className="project--btn">
+                            Source Code
+                        </a>
+                    </button>
+                    <div className="project--technologies-bar">
+                            <div className="bar js-react-notes"></div>
```

Comment:
number of space in the indentation doesn't match.  You have 8 spaces here, but above you have 4. Please keep one standard